extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1706,9 +1706,9 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   renderOpenAttachmentUI_() {
-    // AttachmentEl can be either amp-story-page-attachment or amp-story-page-outlink
+    // AttachmentEl can be any component that extends draggable drawer.
     const attachmentEl = this.element.querySelector(
-      'amp-story-page-attachment, amp-story-page-outlink'
+      'amp-story-page-attachment, amp-story-page-outlink, amp-story-shopping-attachment'
```

Comment:
Would you prefer using the `'amp-story-draggable-drawer-root'` class name in the query selector here and below, for similar reasons as in #36715

Reply:
That sounds good. Since this happens after buildCallback the drawer should already be built and have that class.

Reply:
I think this is a good idea but it makes tests fail since the tags aren't built.

I think this is a good refactor but I'm not sure that this PR is the best place for the refactor since it might have some complexity.

Reply:
Got it, I agree that it's preferable to use your original approach for now